react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react'
+export default function HotButton ({ text }) {
+
+  const [count, setCounter] = useState(0)
+  const [modifier, setModifier] = useState(1)
```

Comment:
Since `modifier` can be calculated from `count`, it doesn't need to be in state. Just move the logic on lines 11-13 into a local variable.